pkg/cmd/stack/deploy.go

```diff
@@ -35,6 +35,9 @@ import (
 
 //Deploy deploys a stack
 func Deploy(ctx context.Context, s *model.Stack, forceBuild, wait bool) error {
+	if s.Namespace != "" {
```

Comment:
I try to initialize the value in the settings struct, but it is private, and it only gets the value from this env variable:
https://github.com/helm/helm/blob/master/pkg/cli/environment.go#L68

Reply:
We are depending on an internal implementation for this. Can we add a test to prevent regressions when updating the helm package?